### DISCUSSION TITLE
Sync listening history for signed in users on webapp

### DISCUSSION THREAD

**Discussion by ajhalili2006** [thechangelog/changelog.com#491] [Ideas]
## The why

Currently, the listening history is stored locally via the browser's `localStorage`, but as someone who switch between devices to listen to podcast episodes on the website while following along the transcript (instead of relying on AI-generated transcripts on Spotify which is currently unavailable on most episodes as far as I know, maybe it's limited to Spotify-hosted shows) is sometimes painful.

## Alternatives

* Sharing timestamped links to self may work, but it may complicate things a lot.
* Since most users will listen to Changelog podcasts on their favorite podcast apps (mine is on Spotify), I think this may be a low priority.

**Comment by jerodsanto** [comment]
Dang somehow I missed this discussion, @ajhalili2006! Sorry about that. I think this is a cool idea and probably not super hard to implement. But you're right, < 1% of our audience listens on the website and I don't see that changing any time soon. 

So, not super high priority but something we're open to!
